src/lightning_app/core/constants.py

```diff
@@ -63,3 +62,9 @@ def get_lightning_cloud_url() -> str:
 
 DEBUG: bool = lightning_cloud.env.DEBUG
 DEBUG_ENABLED = bool(int(os.getenv("LIGHTNING_DEBUG", "0")))
+ENABLE_PULLING_STATE_ENDPOINT = bool(int(os.getenv("ENABLE_PULLING_STATE_ENDPOINT", "1")))
+ENABLE_PUSHING_STATE_ENDPOINT = ENABLE_PULLING_STATE_ENDPOINT and bool(
+    int(os.getenv("ENABLE_PUSHING_STATE_ENDPOINT", "1"))
+)
+ENABLE_STATE_WEBSOCKET = bool(int(os.getenv("ENABLE_STATE_WEBSOCKET", "0")))
+ENABLE_UPLOAD_ENDPOINT = bool(int(os.getenv("ENABLE_UPLOAD_ENDPOINT", "1")))
```

Comment:
As we are growing our catalog of environment variables, we should be thinking of prefixing them with `LIT_` or `LAI_` or similar. For debugging purposes (I can easily grep for the prefix) and to avoid accidental clashes (which could be incredibly difficult to debug).

Reply:
Yes, let's organise them all in another PR.